tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -0,0 +1,412 @@
+{
```

Comment:
<p>Maybe instead of "away from transition", it could be "away from resonance"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>